MAINTAINERS.md

```diff
@@ -0,0 +1,8 @@
+Guidelines for maintainers (those with write access) of rodio. Note these are
+guidelines not rules. Adhere to them as long as makes sense. This is a work in
+progress document, more might follow as we get to know what works. 
+
+Please feel free to open an issue and discuss these if you have a suggestion.
+
+1. Do not merge your own code to main, unless of course its a trivial change.
```

Comment:
IMO such a rule is good if there is a large enough maintainer community behind it. In smaller projects, I think it is less viable. But if there is at least two maintainers where both are well available, why not.